### ISSUE TITLE
Dynamic Switch Between Different Cubemap Formats

### ISSUE THREAD

**Issue by PeterMerkert** [inalogic/pico-pixel-public#20]
Hi, a toggle to switch between different cubemap formats would be great.

Equirectangular
Vertical Cross
Horizontal Cross
Vertical Strip
Horizontal Strip

**Comment by jaytaoko** [comment]
Thank you for reporting. I am looking into this possibility. Does one of the cube map view mode that PicoPixel offers match the Horizontal Cross that you are referring to? 
(Press "Enter" to switch between the view modes for a cubemap texture).

**Comment by PeterMerkert** [comment]
Yes, it does match the horizontal cross one I am referring to. I guess a good resource for the convention is Unity: https://docs.unity3d.com/Manual/class-Cubemap.html

It seems to be another problem here:
1. I load a cube map and it will default to the face view (single face). Arrow up and down to navigate through them.
2. I press "Enter" and I get to the horizontal cross.
3. I press "Enter" and then I get somehow to a big single face view. I can neither scroll out nor in.
4. Third time "Enter" will go back to step 1.

The extension would be really nice! As I work with IBL in game development, I handle cubemaps day in day out.  Anyway, your application is definitely the one of choice for my coworkers and me for any kind of image investigation when handling DXT, BC or raw files. Really, really nice work!

**Comment by jaytaoko** [comment]
Thank you for your feedback. I am working on a new feature to address this.
As for step 3, in that view, you are looking at the cubemap from inside a cube. Click an drag the mouse to look around.
The new feature will allow you to have more control over PicoPixel image rendering.
Regards

**Comment by PeterMerkert** [comment]
Wow, the inside view is really cool! Very convenient also!

The only problematic cubemap format is maybe the 2:1 equirectangular one as you might get false positives. Pictures might be 2:1 ratio, but not cubemaps. But besides that I think the features would be great.
